Projet/render/Moteur.h

```diff
@@ -159,6 +160,7 @@ template <class T, class TDevice> class CMoteur :public Singleton<T>
 	CDIManipulateur& GetGestionnaireDeSaisie() { return GestionnaireDeSaisie; }
 	ResourcesManager& GetResourcesManager() { return ResourcesManager; }
 	Scene& GetScene() { return *CurrentScene; }
+	void SetSkyBox(Pitbull::Actor* Box) { SkyBox = std::unique_ptr<Pitbull::Actor>(std::move(Box)); }
```

Comment:
There is no need to move the Box ptr here

Reply:
look at the addActor in the scene, i did a std::move but i am not sure it is usefull